src/dao/pages/CreateADAO/CreateADAOPage.tsx

```diff
@@ -159,6 +160,7 @@ export function CreateADAOPage() {
         quorum: voting?.quorum ?? 0,
         votingDuration: voting?.duration ?? 0,
         lockingDuration: voting?.lockingPeriod ?? 0,
+        daoFee: govDAOConfig?.daoFee || 0,
```

Comment:
DAO fees should be configurable by the DAO creator. Was there a discussion to default the `daoFee` to the factory contract configuration fee due to time constraints?

Reply:
@damiano-melcarne-lab49 this is DAO creation fee. As my discussion with @roshan-bisht-lab49  it will be fixed in the config level. Proposal creation fee will be configurable by the DAO creator or via proposal.